docs/getting-started/configuration/crossplane.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: meta.pkg.crossplane.io/v1
+kind: Configuration
+metadata:
+  name: getting-started-with-aws
+  annotations:
+    uxp-guide: getting-started
+spec:
+  crossplane:
+    version: ">=v1.2.1"
+  dependsOn:
+    - provider: crossplane/provider-aws
+      version: "v0.18.1"
```

Comment:
are we pinning to an exact version of provider-aws for a particular reason?

Reply:
It's safer. I actually think the upstream Crossplane version of this same config might be broken, because it wanted `>=v0.16.0` but I had to update the `Composition` after we made breaking changes to an alpha resource (bumping it to v1beta1).

Pinning down to the patch version is probably too extreme, but I've already pushed this so I'd prefer to leave it as is.